spec/tbd_tests_spec.rb

```diff
@@ -2222,13 +2273,14 @@
     expect(door_trns_m).to be_within(0.01).of(maxZ - 0.9 + maxY - minY)
 
     io[:edges].each do |edge|
-      expect(edge.key?(:surfaces))
-      expect(edge[:surfaces].is_a?(Array))
-      expect(edge[:surfaces].empty?).to be(false)
+      expect(edge.key?(:surfaces)        ).to be( true)
+      expect(edge[:surfaces].is_a?(Array)).to be( true)
+      expect(edge[:surfaces].empty?      ).to be(false)
+
       next unless edge[:surfaces].include?(trnsom)
 
-      expect(edge.key?(:type  ))
-      expect(edge.key?(:length))
+      expect(edge.key?(:type  )).to be(true)
```

Comment:
One of many (!) examples where the RSpec expectations were incomplete (? more like MiniTest asserts?). While fixing, the spec file was reformatted to facilitate visual assessment of code completion. The vast majority of committed changes deal strictly with formatting.